GOVERNANCE.md

```diff
@@ -0,0 +1,147 @@
+project. However, such contributions must be approved by a project
+Committer before they become a part of the project.  
+
+Anyone can be a Contributor. You need no formal approval from the
+project, beyond the legal forms.
+
+### How to Become a Contributor
+
+* Review the coding standards in `CONTRIBUTING.md` to ensure your contribution
+  is in line with the project's coding and styling guidelines.
+
+* Sign the Individual CLA, or have your organization sign the Corporate CLA.
+
+* Submit your code as a PR with the appropriate DCO sign-off.
+
+## Committers
+
+Project Committers have merge access on the Rez GitHub repository
+and are responsible for approving submissions by Contributors.
+
+### Committer Responsibilities
+
+Typical activities of a Committer include:
+
+* Helping users and novice contributors.
+
+* Ensuring a response to questions posted to the ASWF rez slack channel.
+
+* Contributing code and documentation changes that improve the project.
+
+* Reviewing and commenting on issues and pull requests.
+
+* Ensuring that changes and new code meet acceptable standards and are
+  in the long-term interest of the project.
+
+* Participation in working groups.
+
+* Merging pull requests.
+
+### How to Become a Committer
+
+Any existing Committer can nominate an individual making significant
+and valuable contributions to the Rez project to become a new
+Committer. New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at:
+  rez-discussion@lists.aswf.io
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](https://github.com/AcademySoftwareFoundation/foundation/blob/main/project_charters/rez-charter.pdf).
+This charter defines the TSC member terms and succession policies.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC. The Chair may
+also serve as the Rez representative on the Academy Software
+Foundation (ASWF) Technical Advisory Council (TAC), or one may
+be separately elected by the TSC. The Rez representative will
+represent the project at ASWF TAC meetings.
+
+### Current TSC Members
+
+Unless otherwise denoted with †, members of the TSC contribute to rez
```

Comment:
Based on https://tac.aswf.io/process/tsc_faq.html#q-does-an-individual-serving-on-the-tsc-represent-themselves-the-project-andor-their-employer, TSC members shouldn't act on behalf of their employer:

> As TSC members are individuals within the context of a TSC, it’s expected that all their actions are for the general good of the project itself. It’s not appropriate to use the project for anti-competitive purposes, nor is it appropriate to discuss a TSC member employer’s product plans and roadmap in conjunction with the project planning.

The last bit is important, "It’s not appropriate to use the project for anti-competitive purposes, nor is it appropriate to discuss a TSC member employer’s product plans and roadmap in conjunction with the project planning".

I think we should be clear on that. Right now the phrasing kind of lets the reader think that a TSC member could potentially act on behalf of its employer in the future.

Oh,, and I would denitiviely keep "but are listed regardless for transparency".

Reply:
I understand your statement, but am not sure what sort of statement you would append to provide additional clarity. Do you want us to link back to the TSC FAQ or copy-paste that statement or something else?

Reply:
How about:

> Members of the TSC contribute to rez independent of their employer and their employer is listed purely for transparency purposes.

Short and to the point.

Reply:
replaced.